CalendarApp/NUnitTestProject/EventManagerTests/EventManager.MonthlyEventTest.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using BookingRoom.Helpers;
+using BookingRoom.Models.GoogleEvent;
+using Google.Apis.Calendar.v3.Data;
+using NUnit.Framework;
+
+namespace NUnitTestProject.EventManagerTests
+{
+    [TestFixture]
+    public class MonthlyEventConflictsTest : EventManagerTestClass
+    {
+        [SetUp]
+        public void SetUp()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            var conflictedEvent = _eventFactory.CreateEvent(
+                new EventTime(2015, 4, 19, 10, 0, 0), new EventTime(2015, 4, 19, 12, 0, 0),
+                "Nunit", "NunitDecription");
+            _meetingBooking.PostEvent(ToEventConverter.ToEvent(conflictedEvent, _timezone), calendarId);
+
+        }
+
+        [TearDown]
+        public void TearDown()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+            var eventList = _meetingBooking.GetEvents(calendarId);
+            foreach (var e in eventList)
+            {
+                _calendarService.Events.Delete(calendarId, e.Id).Execute();
+            }
+        }
+
+        [Test]
+        public void EventManager_MonthlyRecurrent_FindConflicts_InterSectionOneDay()
+        {
+            var calendarId = AppSettingsHelper.GetAppSetting(AppSetingsConst.TestCalendar);
+
+            var eventForAdd = _eventFactory.CreateEvent(
+              new EventTime(2015, 1, 19, 10, 0, 0), new EventTime(2015, 1, 19, 15, 0, 0),
+              "Nunit", "NunitDecription");
+            eventForAdd.Recurrence = new[] { "RRULE:FREQ=DAILY;UNTIL=20151219T100000Z" };
```

Comment:
this strings are reused. they can be converted to a class level constants or wraped as return value of the specific method in test helper class